articles/openshift/oshift-how-expand-persistent-vols.md

```diff
@@ -0,0 +1,162 @@
+
+In OpenShift clusters running OpenShift v3.11 or later, it is possible to expand existing persistent volumes. It is necessary to shutdown any pod/container using the volume before the expansion can be performed. In earlier versions (such as v3.9) it is necessary to raise a support request so that UKCloud admins can facilitate the expansion.
+
+>[!TIP]
+>To check the version of your cluster, either click "(?)" in the top-right of the web UI and select "About" or run the following while connected in the CLI client:
+>```
+>$ oc version
+>...
+>Server https://console.5678-89abcd:8443
+>openshift v3.11.51
+>kubernetes v1.11.0+d4cacc0
+>```
+
+## Prerequisites
+
+To complete the steps in this guide, you must have access to a cluster that is running OpenShift Container Platform v3.11 (or newer).
+
+You must also have access to:
+
+- `oc`, the OpenShift command-line client (CLI). For more information, see OpenShift's [*Get Started with the CLI*](https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html)
+
+If the cluster is v3.11 or newer, the storage class should be enabled for Volume Expansion - you can confirm this as follows:
+```
+$ oc describe storageclass tier2
+Name:                  tier2
+IsDefaultClass:        Yes
+Annotations:           storageclass.beta.kubernetes.io/is-default-class=true
+Provisioner:           kubernetes.io/cinder
+Parameters:            availability=nova,type=TIER2
+AllowVolumeExpansion:  True
+MountOptions:          <none>
+ReclaimPolicy:         Delete
+VolumeBindingMode:     Immediate
+Events:                <none>
+```
+
+## Stop any pod which is using the volume.
+
+>[!INFO]
+>In this guide, we will expand the volume for cluster's default elasticsearch deployment which is located in the `openshift-logging` project. The pod using the persistent volume in this example is `logging-es-data-master` and the pvc (Persistent Volume Claim) name is `logging-es-0`. This is a common requirement - a large volume of logs may cause elasticsearch to fail when the logging volume fills up.
+>
+>The same procedure can be applied to any other pods which use persistent volumes.
+
+```
+$ oc project openshift-logging
+Now using project "openshift-logging" on server "https://console.5678-89abcd:8443".
+$ oc get pods
+NAME                                      READY     STATUS      RESTARTS   AGE
+logging-es-data-master-7bqadxxd-1-wrlqh   2/2       Running     2          2d
+...
+logging-kibana-1-n7j8t                    2/2       Running     2          2d
+$ oc get dc
+NAME                              REVISION   DESIRED   CURRENT   TRIGGERED BY
+logging-es-data-master-7bqadxxd   1          1         1
+logging-kibana                    1          1         1         config
+```
+>[!INFO] In this example, the `logging-kibana` pod depends on the `logging-es-data-master` pod so it is good practice to also scale this down during the expansion of the persistent volume for `logging-es-data-master`
+```
```

Comment:
Could we replace this example section with something similar to the below?

```
for i in $(oc get dc | awk 'NR>1 {print $1}') ; do oc scale --replicas=0 dc $i; done
deploymentconfig.apps.openshift.io/logging-kibana scaled
deploymentconfig.apps.openshift.io/logging-es-data-master-7bqadxxd scaled
```

This reduces the requirement to change the guide to match the es dc name